docs/recipes/browser-testing.md

```diff
@@ -38,6 +38,15 @@ import browserEnv from 'browser-env';
 browserEnv(['window', 'document', 'navigator']);
 ```
 
+If your components use jQuery, you can make it available like so:
```

Comment:
I wonder if you could rephrase this a little. What's interesting is that you're adding additional global variables. jQuery could be an example of that:

> You can expose more global variables by assigning them to the `global` object. For instance jQuery is typically available through the `$` variable: